### ISSUE TITLE
Installation Issue: Windows not on C:

### ISSUE THREAD

**Issue by GoogleCodeExporter** [prajaktamoghe/selenium-vba#4]
```
When running SeleniumWrapperSetup-1.0.5.0.exe I got this error:

"
Unable to execute file:
C:\WINDOWS\Microsoft.NET\Framework\v2.0.50727\RegAsm.exe

CreateProcess failed; code 3.
The system cannot find the path specified.
"

On the computer I was installing the wrapper to, Windows is on drive D. I 
created a junction to the windows folder in the root of C: and tried installing 
again. This made it work.

If you're curious on how I made a junction on windows xp, see the following:
http://schinagl.priv.at/nt/hardlinkshellext/hardlinkshellext.html
```

Original issue reported on code.google.com by `matthewkastor@gmail.com` on 8 May 2012 at 10:31

**Comment by GoogleCodeExporter** [comment]
```
Fixed in version 1.0.5.1

```

Original comment by `florentbr` on 9 May 2012 at 7:13
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Awesome! Thank you. :D
```

Original comment by `matthewkastor@gmail.com` on 16 May 2012 at 10:36
